### PR TITLE
Moved midi, synthesizers, drives, exports to audio module

### PR DESCRIPTION
**Pull request by igorkorsukov**
To make the code more structured, I moved midi, synthesizers, drives, exports to to a separate module.

- [x] I signed [CLA](https://musescore.org/en/cla)
- [x] I made sure the code in the PR follows [the coding rules](https://musescore.org/en/handbook/developers-handbook/finding-your-way-around/musescore-coding-rules)
- [x] I made sure the code compiles on my machine
- [x] I made sure there are no unnecessary changes in the code
- [x] I made sure the title of the PR reflects the core meaning of the issue you are solving
- [x] I made sure the commit message(s) contain a description and answer the question "Why do those changes fix that particular issue?" or "Why are those changes really necessary as improvements?"
- [x] I made sure the commit message title starts with "fix #424242:" if there is a related issue

### CODE REVIEW

File: audio/CMakeLists.txt [hunk 1]
```diff
@@ -31,7 +31,7 @@ include_directories (
       ${CMAKE_CURRENT_BINARY_DIR}
       ${CMAKE_CURRENT_LIST_DIR}
       ${PROJECT_SOURCE_DIR}/global
-      ${PROJECT_SOURCE_DIR}/thirdparty # because #include "mscore/musescore.h"
+      ${PROJECT_SOURCE_DIR}/thirdparty  #because #include "mscore/musescore.h"
```
Comment (Jojo-Schmitz) [inline audio/CMakeLists.txt:34]: A `git commit --amend` save the commit message unchanged and `git push -f` would have do that too ;-)

Comment (igorkorsukov) [reply audio/CMakeLists.txt:34]: Thanks. 
I knew that there would be conflicts after the merger of another PR, so I just made it a separate commit :)

Comment (Jojo-Schmitz) [reply audio/CMakeLists.txt:34]: And indeed a rebase is needed now ;-)

Comment (Jojo-Schmitz) [reply audio/CMakeLists.txt:34]: Also it didn't help, Travis still failed

**Pull request closed without merging**